### DISCUSSION TITLE
Ability to choose which sites entry types are available in

### DISCUSSION THREAD

**Discussion by chrismlusk** [craftcms/cms#12866] [Ideas] [answered]
In a multisite setup, it would be nice to be able to configure site-specific settings for Entry Types.

**Comment by brandonkelly** [comment, marked as answer]
This will sortof be possible in Craft 5, as entry types are becoming decoupled from sections. (So sections will choose which entry types are available to it, rather than actually defining their entry types).

With that in place, you would be able to set up two separate sections—one available for Site A; the other available for Site B—which could share some entry types, but not others.

**Reply by chrismlusk** [reply]
That makes sense, but as you say, it is only sort of a solution. Having that ability would certainly help, but creating separate sections does feel a bit odd in certain situations — like with a generic `Pages` section used in many projects.

Until Craft 5, though, if I want to customize Entry Type options to prevent editors in a specific site from accidentally selecting an entry type that isn't supported for them, any suggestions? Would `DefineEntryTypesEvent` be an option?

**Reply by brandonkelly** [reply]
I don’t think limiting entry types to certain sites within the same section would be possible. It would create a lot of complications for site propagation.
